AsyncDisplayKit/ASMultiplexImageNode.h

```diff
@@ -195,7 +195,7 @@ didFinishDownloadingImageWithIdentifier:(id)imageIdentifier
  * @abstract An image URL for the specified identifier.
  * @param imageNode The sender.
  * @param imageIdentifier The identifier for the image that will be downloaded.
- * @discussion Supported URLs include assets-library, Photo framework URLs (ph://), HTTP, HTTPS, and FTP URLs.  If the
+ * @discussion Supported URLs include assets-library, URLs converted from ASPhotosImageRequest, HTTP, HTTPS, and FTP URLs.  If the
```

Comment:
That's my bad I probably could have explained it more clearly. Photos.framework URLs aren't really "a thing," it's something made-up so the only way to form/parse them is through ASPhotosImageRequest. So I expect the typical use pattern will be 1. create and configure an ASPhotosImageRequest using an asset ID & target size 2. return its URL 3. ASMultiplexImageNode will attempt to parse the URL into an ASPhotosImageRequest like you mentioned